packages/twenty-server/src/engine/metadata-modules/field-metadata/validators/is-field-metadata-options.validator.ts

```diff
@@ -42,6 +42,12 @@ export class IsFieldMetadataOptions {
       try {
         fieldMetadata = await this.fieldMetadataRepository.findOneOrFail({
           where: { id },
+          relations: [
+            'fromRelationMetadata',
+            'toRelationMetadata',
```

Comment:
same here, we never do
fieldMetadata.fromRelationMetadata and such in this file with this fieldMetadata instance I think